gst/edge/edge_common.c

```diff
@@ -31,6 +31,8 @@ gst_edge_get_connect_type (void)
           "Connect with MQTT brokers and directly sending stream frames via TCP connections."},
       {NNS_EDGE_CONNECT_TYPE_AITT, "AITT",
           "Sending stream frames via AITT connections."},
+      {NNS_EDGE_CONNECT_TYPE_AITT, "MQTT",
```

Comment:
How about replacing 'NNS_EDGE_CONNECT_TYPE_AITT' with 'NNS_EDGE_CONNECT_TYPE_MQTT' ?

Reply:
nnstreamer-edge supports both AITT and MQTT pub/sub.